tests/integration/interactive_deploy_test.go

```diff
@@ -159,20 +159,8 @@ var _ = Describe("odo deploy interactive command tests", func() {
 			It("should not fail but fallback to the interactive mode", func() {
```

Comment:
I just checked and it seems we have this same test with the PHP stack for `odo init`: https://github.com/redhat-developer/odo/blob/main/tests/integration/interactive_init_test.go#L655-L672. I guess those should have failed as well. Can we do the same thing there as well?